### DISCUSSION TITLE
Welcome to mnemocards!

### DISCUSSION THREAD

**Discussion by guiferviz** [guiferviz/mnemocards#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Otter-man** [comment]
Okay, here comes my late introduction.

Hi, I'm @Otter-man I'm a coder and an ardent student of things. 
After learning about spaced repetition in 2018 via [Gwern's post](https://www.gwern.net/Spaced-repetition) and [Nicky Case's amazing introduction comics/game](https://ncase.me/remember/) I started to use it myself and gradually became a true believer in the system.

Once I decided to brush on my English vocabulary (which is my second language) I looked for a tool to generate vocabulary cards automatically (including auto translation). There were none at the time, so I had to write a tool myself. But during the search, I saw a post with the introduction of Mnemocards on Reddit and noted to myself to maybe adapt my code to this card generator in the future.

I was still new to coding at the time and was thinking of an old adage "want to improve your coding skills - join open source project". Which I did at the start of last year by porting my automatic vocabulary generator and creating a new builder in Mnemocards.  
As far as I know, Mnemocards is still the only tool that has this functionality and also does it in style.

Since then I follow the project and try to be of help. I turn to Mnemocards during times when I feel like I don't have anything going on but want to be productive and do something cool. 

I'm glad you have a vision for Mnemocards v 1.0  and would like to help build it.
